db/schema.rb

```diff
@@ -15,11 +15,24 @@
   # These are extensions that must be enabled in order to support this database
```

Comment:
The above migration was one that wei-yee created I believe. For some reason it was never reflected in the schema, probably due to some git merge problem so I renamed the above migration and migrated it as well. The schema below should be correct and reflect all migrations thus far.